### PR TITLE
feat: Scroll deployments

### PR DESCRIPTION
**Pull request by pxrl**

### CODE REVIEW

File: deploy/consts.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const L1_ADDRESS_MAP: { [key: number]: { [contractName: string]: string }
     usdc: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
     scrollERC20GatewayRouter: "0xF8B1378579659D8F7EE5f3C929c2f3E332E41Fd6",
     scrollMessengerRelay: "0x6774Bcbd5ceCeF1336b5300fb5186a12DDD8b367",
-    scrollGasPriceOracle: "0x987e300fDfb06093859358522a79098848C33852",
+    scrollGasPriceOracle: "0x0d7E906BD9cAFa154b048cFa766Cc1E54E39AF9B",
```
Comment (pxrl) [inline deploy/consts.ts:33]: The existing mainnet gas price oracle was migrated in ~February: https://docs.scroll.io/en/developers/scroll-contracts/#advanced-bridge-contracts

File: deploy/consts.ts [hunk 2]
```diff
@@ -254,6 +254,12 @@ export const L2_ADDRESS_MAP: { [key: number]: { [contractName: string]: string }
     scrollGasPriceOracle: "0x5300000000000000000000000000000000000002",
     scrollMessenger: "0xba50f5340fb9f3bd074bd638c9be13ecb36e603d",
   },
+  534352: {
+    l2Weth: "0x5300000000000000000000000000000000000004",
+    scrollERC20GatewayRouter: "0x4C0926FF5252A435FD19e10ED15e5a249Ba19d79",
+    scrollGasPriceOracle: "0x5300000000000000000000000000000000000002",
+    scrollMessenger: "0x781e90f1c8Fc4611c9b7497C3B47F99Ef6969CbC",
+  },
```
Comment (pxrl) [inline deploy/consts.ts:262]: Contract addresses defined here: https://docs.scroll.io/en/developers/scroll-contracts/

**Review by dohaki — APPROVED** [review]
LGTM

**Review by bmzig — APPROVED** [review]
Double checked the contract addresses. They check out.

**Pull request merged**